extensions/binding/org.eclipse.smarthome.binding.onewire/README.md

```diff
@@ -0,0 +1,183 @@
+The correct formula for the ambient light is automatically determined from the sensor version.
+
+## Channels
+
+| Type-ID         | Thing                       | Item    | readonly   | Description                                        |
+|-----------------|-----------------------------|----------------------|----------------------------------------------------|
+| current         | multisensors                | Number  | yes        | current (if light option not installed)            |
+| digital         | digitalX, AMS               | Switch  | no         | digital, can be configured as input or output      |
+| humidity        | multisensors (except ms-tv) | Number  | yes        | relative humidity                                  |
+| light           | ams, bms                    | Number  | yes        | lightness (if installed)                           |
+| present         | all                         | Switch  | yes        | sensor found on bus                                |
+| supplyvoltage   | multisensors                | Number  | yes        | sensor supplyvoltage                               |
+| temperature     | not digitalX, ibutton       | Number  | yes        | environmental temperature                          |
+| voltage         | ms-tv, ams                  | Number  | yes        | voltage input                                      |
+
+### Digital I/O (`digitalX`)
+
+The `digitalX` channels each have two parameters: `mode` and `logic`.
+
+The `mode` parameter is used to configure this channels as `input` or `output`.
+
+The `logic` parameter can be used to invert the channel.
+In `normal` mode the channel is considered `ON` for logic high, and `OFF` for logic low.
+In `inverted` mode `ON` is logic low and `OFF` is logic high.
+
+### Humidity (`humidity`)
+
+Depending on the sensor, the `humidity` channel may have the `humiditytype` parameter.
+This is only needed for the `ms-th` sensors.
+`ams` and `bms` sensors select the correct sensor type automatically.
+
+Possible options are `/humidity` for HIH-3610 sensors, `/HIH4000/humidity` for HIH-4000 sensors, `/HTM1735/humidity` for HTM-1735 sensors and `/DATANAB/humidity` for sensors from Datanab.
+
+### Temperature (`temperature`)
+
+The `temperature` channel has two parameters: `resolution` and `offset`.
+
+OneWire temperature sensors are capable of different resolutions: `9`, `10`, `11` and `12` bits.
+This corresponds to 0.5 °C, 0.25 °C, 0.125 °C, 0.0625 °C respectively.
+The conversion time is inverse to that and ranges from 95 ms to 750 ms.
+For best performance it is recommended to set the resolution only as high as needed.
+
+The second parameter is the `offset`.
+Onewire sensors are usually well calibrated but depending on the mounting position of the sensor it may be necessary to apply an offset.
+The `offset` parameter consists of the value itself and it's unit (e.g. `-1.0 °C`).
+ 
+ 
+## Full Example
+
+text
+
+### demo.things:
```

Comment:
It would be nice to also see a more complex example, maybe with additional channel options and multiple sensor ids.

Reply:
I'll have a look at that. Since I don't use it, I'll have to figure out how this works in textual configuration.